inc/components/text/component.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Text.
+ *
+ * Concatenate and render a string.
+ *
+ * @package Irving_Components
+ */
+
+namespace WP_Irving;
+
+use WP_Irving\Component;
+
+if ( ! function_exists( '\WP_Irving\get_registry' ) ) {
+	return;
+}
+
+/**
+ * Register the component and callback.
+ */
+get_registry()->register_component_from_config( __DIR__ . '/component' );
+
+/**
+ * Output the content config value as a text node instead of a component.
+ *
+ * @todo Deterine how text and strings should actually work, and target
```

Comment:
Let's not forget this.

Reply:
It's on the top of my mind. Let's discuss tomorrow.